dev/test/specs/starter-test.js

```diff
@@ -216,7 +216,9 @@ function allSuggestions() {
 }
 
 function acceptSuggestion(textSubstring) {
+  wait(2);
```

Comment:
Is this required? openSuggestionDrawer() also starts with wait(2), right?

Reply:
trying and wondering led to this :(
maybe i should increase the wait time in waitForStillness instead?
i could try it.

even reducing the wait here to 1 seconds causes the test (more consistently) to fail...